exchange.go

```diff
@@ -442,7 +475,144 @@ func (re *RabbitExchangeImpl) Receive(exchange ExchangeSettings, queue QueueSett
 						closer()
 						for {
 							time.Sleep(time.Millisecond * 250)
-							channel, msgs, errChan, closer, err = getChannel()
+							channel, msgs, errChan, closer, err = re.getChannel(exchange, queue, retryExchangeName)
```

Comment:
Basically, time.Sleep should not be preferred in most of the application.
I really couldn't  get that article which I read before. Hope this article gives the minimum detail
https://stackoverflow.com/questions/9154899/the-disadvantages-of-using-sleep.

If we use the time.Sleep, it's not completely sure about the time needs to be sleep and also it completely sleeps the entire process in the CPU to sleep.

We should use a different way to re-trigger it like using `time.NewTIcker()` .

Reply:
Will add a todo for that, good suggestion. It is however outside the scope of this and I would like to avoid breaking unnecessary things D: